src/Umbraco.Web.UI.Client/src/views/propertyeditors/slider/handle.prevalues.html

```diff
@@ -0,0 +1,10 @@
+﻿<div>
+    <select ng-model="model.value" required name="handle">
+        <option value="round">Round</option>
+        <option value="square">Square</option>
+        <option value="triangle">Triangle</option>
+        <!--<option value="custom">Custom</option>-->
```

Comment:
For now I don't add the option for using an icon as custom handle, e.g. a "star".